src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -52,8 +55,11 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
     studentParticipation: StudentParticipation | null;
     isAfterAssessmentDueDate: boolean;
     public gradingCriteria: GradingCriterion[];
-
+    private programmingExercise: ProgrammingExercise;
+    public wasSubmissionSimulated = false;
```

Comment:
please separate this with a section from the real "production" code

Reply:
!

Reply:
i separated the variables